multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -120,6 +120,17 @@ def ocr_text_detection():
     }
 
 
+@automm_presets.register()
+def ocr_text_recognition():
+    return {
+        "model.names": ["mmocr_text_recognition"],
+        "model.mmdet_image.checkpoint_name": "ABINet",
```

Comment:
Should it be `"model. mmocr_text_recognition.checkpoint_name": "ABINet"`?

Reply:
Done.